### PR TITLE
Fixed Crypto Transactions failures

### PR DESCRIPTION
**Pull request by zubair-deriv**
### Card:
https://app.clickup.com/t/20696747/WALL-4331

### CODE REVIEW

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 1]
```diff
@@ -75,13 +75,15 @@ describe('QATEST-98789 - Transfer to crypto accounts and QATEST-98794 View Crypt
     cy.findAllByText(/USD Wallet/)
       .first()
       .should('be.visible')
-    cy.findAllByText(`-${transferAmount}`).first().should('be.visible')
+    cy.contains(`-${transferAmount}`).first().should('be.visible')
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:78]: Why this change?

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 2]
```diff
@@ -75,13 +75,15 @@ describe('QATEST-98789 - Transfer to crypto accounts and QATEST-98794 View Crypt
...
   })
 
   it('should be able to perform transfer from crypto account in responsive', () => {
     cy.log('Transfer from Crypto account')
     cy.c_visitResponsive('/', 'small')
+    cy.c_skipPasskeysV2()
     cy.contains('Wallet', { timeout: 10000 }).should('exist')
+    cy.c_skipPasskeysV2()
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:86]: Why calling twice?

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 3]
```diff
@@ -93,7 +95,9 @@ describe('QATEST-98789 - Transfer to crypto accounts and QATEST-98794 View Crypt
   it('should be able to view transactions of crypto account in responsive', () => {
     cy.log('View Transactions of Crypto account')
     cy.c_visitResponsive('/', 'small')
+    cy.c_skipPasskeysV2()
     cy.contains('Wallet', { timeout: 10000 }).should('exist')
+    cy.c_skipPasskeysV2()
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:100]: Same ques as above.

**Review by abuzar-deriv — COMMENTED** [review]
A couple of questions.

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js (hunk 3 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:100]: Because this Passkeys modal was appearing twice

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js (hunk 1 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:78]: Amount was findable by cypress using cy.findAllByText(), so I changed it to cy.contains()

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js (hunk 2 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js]: removed the 1st method, the single passkeys method works fine once a dom element is found

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js (hunk 3 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js]: removed the 1st method,

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by afsaneh-rezaeinejad-deriv — APPROVED** [review]
